### PR TITLE
[8.13] add truncated_stack to api.behaviors documentation

### PR DESCRIPTION
**Pull request by jdu2600**
## Change Summary

Updated documentation for `process.Ext.api.behaviors` for new value since 8.13.3

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes

### CODE REVIEW

**Comment by pzl** [comment]
We can merge this, but FYI we are not expecting any more 8.13 package releases.

Is this something you want a release specifically cut for? Otherwise the change will likely never make it out into the wild

**Comment by jdu2600** [comment]
It's a very minor documentation-only update - and having it appear in 8.14 should be fine.
cc: @nfritts @magermark

**Comment by pzl** [comment]
8.14 is also released and final, this would be 8.15

**Review by intxgo — APPROVED** [review]

**Review by szwarckonrad — APPROVED** [review]

**Comment by jdu2600** [comment]
> 8.14 is also released and final, this would be 8.15

Ah. Noted.
Not ideal but not a significant issue either.

File: package/endpoint/data_stream/api/fields/fields.yml [hunk 1]
```diff
@@ -1104,7 +1104,7 @@
...
...mory\n  \"execute_fluctuation\" - the PAGE_EXECUTE protection is unexpectedly fluctuating\n  \"write_fluctuation\" - the PAGE_WRITE protection of executable memory is unexpectedly fluctuating\n  \"hook_api\" - a change to the memory protection of a small executable image memory region was made\n  \"hollow_image\" - a change to the memory protection of a large executable image memory region was made\n  \"hook_unbacked\" - a change to the memory protection of a small executable non-image memory was made\n  'hollow_unbacked\" - a change to the memory protection of a large executable non-image memory was made\n  \"guarded_code\" - executable memory was unexpectedly marked as PAGE_GUARD\n  \"hidden_code\" - executable memory was unexpectedly marked as PAGE_NOACCESS\n  \"execute_shellcode\" - a region of non-image executable memory was unexpectedly transferred control\n  \"hardware_breakpoint_set\" - a hardware breakpoint was set\n  \"rapid_background_polling\" - a suspicious process which does rapid input polling via GetAsyncKeyState API was observed\n  \"multiple_polling_processes\" - multiple suspicious processes which do rapid input polling via the GetAsyncKeyState API were observed"
```
Comment (magermark) [inline package/endpoint/data_stream/api/fields/fields.yml:1107]: Any particular reason why characters are escaped in this description? That doesn't seem to be the case within this yml file

File: schemas/v1/api/api.yaml [hunk 2]
```diff
@@ -2290,16 +2290,17 @@ process.Ext.api.behaviors:
...
-    \ executable memory\n  \"execute_fluctuation\" - the PAGE_EXECUTE protection is\
-    \ unexpectedly fluctuating\n  'write_fluctuation\" - the PAGE_WRITE protection\
-    \ of executable memory is unexpectedly fluctuating\n  \"hook_api\" - a change\
-    \ to the memory protection of a small executable image memory region was made\n\
-    \  \"hollow_image\" - a change to the memory protection of a large executable\
-    \ image memory region was made\n  \"hook_unbacked\" - a change to the memory protection\
-    \ of a small executable non-image memory was made\n  'hollow_unbacked\" - a change\
-    \ to the memory protection of a large executable non-image memory was made\n \
-    \ \"guarded_code\" - executable memory was unexpectedly marked as PAGE_GUARD\n\
+    truncated_stack\" - call stack is unexpected truncated due to malicious tampering\
```
Comment (magermark) [inline schemas/v1/api/api.yaml:2293]: Same comment as above re: escaping characters

File: package/endpoint/data_stream/api/fields/fields.yml (hunk 1 above)
Comment (jdu2600) [reply package/endpoint/data_stream/api/fields/fields.yml:1107]: 🤷 Those files are auto-generated.

**Pull request merged**